ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -52,24 +53,30 @@
     - zulu7                       # JDK8 Build Bootstrap
     - role: adoptopenjdk_install
       jdk_version: 8
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK11 Build Bootstrap
       jdk_version: 10
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install
       jdk_version: 11
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK14 Build Bootstrap
       jdk_version: 13
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK15 Build Bootstrap
       jdk_version: 14
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - OpenSSL102                  # OpenJ9
     # - Nagios_Plugins              # AdoptOpenJDK Infrastructure
     # - Nagios_Master_Config        # AdoptOpenJDK Infrastructure
     # - Nagios_Tunnel               # AdoptOpenJDK Infrastructure
-    - riscv_cross_compiler        # For building JDK11/J9 on RISC-V architecture
+    - role: riscv_cross_compiler    # For building JDK11/J9 on RISC-V architecture
+      when: ansible_architecture != "riscv64"
```

Comment:
The role already limits itself to certain platforms which will mean it doesn't install on riscv64 anyway so this is not needed